xep-0425.xml

```diff
@@ -84,30 +96,28 @@
 </message>]]></example>
 
   <p>The moderator sees the message and indicates to their client that the message should be retracted. The client then sends an IQ stanza to the MUC service, requesting that the message be retracted.</p>
+  <p>An optional &lt;reason/&gt; element may be included inside the &lt;moderate/&gt; element, to provide a human-readable explanation why the message was retracted.</p>
 
   <example caption="The moderator's client sends an IQ stanza to the MUC service"><![CDATA[
 <iq type='set' to='room@muc.example.com' id='retract-request-1'>
-  <apply-to id="stanza-id-1" xmlns="urn:xmpp:fasten:0">
-    <moderate xmlns='urn:xmpp:message-moderate:0'>
-      <retract xmlns='urn:xmpp:message-retract:0'/>
-      <reason>This message contains inappropriate content for this forum</reason>
-    </moderate>
-  </apply-to>
+  <moderate id="stanza-id-1" xmlns='urn:xmpp:message-moderate:1'>
+    <retract xmlns='urn:xmpp:message-retract:1'/>
+    <reason>This message contains inappropriate content for this forum</reason>
+  </moderate>
 </iq>]]></example>
 
   <section2 topic='Success case' anchor='usecase-success'>
 
   <p>If the moderator is allowed to moderate the message, the groupchat service will send a message stanza to all participants (including the moderator), indicating that the message has been retracted and by whom.</p>
-  <p>This message will use &xep0422; to indicate that it applies to the moderated message, by referring to its XEP-0359 Stanza ID.</p>
+  <p>This message will use &xep0422; to indicate that it is a retraction, and will refer to the retracted message by its XEP-0359 Stanza ID. Inside the &lt;retract/&gt; element is a &lt;moderated/&gt; element indicating that the retraction was a moderator action and containing the optional reason.</p>
```

Comment:
A reference to a xep359 stanza-id should always use the 'id' *and* the 'by' attribute (not just the 'id'). See https://mail.jabber.org/pipermail/standards/2023-February/039168.html

Reply:
Doesn't look to me like there's uniformity on this. I've added more text to the XEP to clarify that the `by` attribute of the original stanza-id should be from the MUC.